Carrot-Assistant/app.py

```diff
@@ -184,6 +201,7 @@ async def run_pipeline(request: PipelineRequest) -> EventSourceResponse:
     """
     return EventSourceResponse(generate_events(request))
 
+
 @app.post("/run_db")
 async def run_db(request: PipelineRequest) -> dict:
```

Comment:
Can you change the return type?

Reply:
async def run_db(request: PipelineRequest) -> List[Dict[str, Any]]:

Is this return type correct?